invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -136,6 +139,14 @@ export const RDMRecordSearchBarContainer = () => {
   );
 };
 
+const headerSearchbar = document.getElementById("header-search-bar");
+
+export const RDMRecordMultipleSearchBarElement =
+  MultipleOptionSearchBarElementHOC({
+    options: JSON.parse(headerSearchbar.dataset.options),
```

Comment:
`JSON.parse` was happening also in your PR in `invenio-search-ui`, might to be removed there?